UserInputs/userInputsData.py

```diff
@@ -5,6 +5,7 @@ class UserInputsData:
     application = "Village Health"
     domain = "qa-automation"
     download_path = Path('~/Downloads').expanduser()
+    download_path_ci = r"/home/runner/work/dimagi-qa/dimagi-qa/"
```

Comment:
I think things would be easier if you did the CI check here:
```suggestion
    if os.environ.get("CI") == "true":
        download_path = Path("/home/runner/work/dimagi-qa/dimagi-qa/")
    else:
        download_path = Path('~/Downloads').expanduser()
```
That way you can just reference `UserInputsData.download_path` everywhere else without having to repeat that check.